eng/Version.Details.xml

```diff
@@ -129,6 +129,10 @@
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>95baa7ae2dca00a29a68b5bad14d8331dd5dc570</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.NETCore.App.Internal" Version="5.0.0-rc.1.20430.6" CoherentParentDependency="Microsoft.Private.Winforms">
```

Comment:
This will work for .NET 5, but we'll need to find a different solution for .NET 6 since we're planning on removing the Microsoft.NETCore.App.Internal package.

Reply:
It can be any **non-stable** package that revs on every release. Why is this being removed though? Is it being removed for each repo (each of the runtime repos generates one).

Reply:
This package was produced as a compat shim for the .NET Core 2.0-style way of referencing the shared framework. From what I've been able to discover, the only usages of this package outside of dotnet/runtime are for this exact usage (getting a non-stable package version) and with the move to the new shared framework SDK, we're moving away from our internal usage of this package so we can actually test the product the way our users consume it (framework packs instead of NuGet packages)

Reply:
We can keep an empty Microsoft.NETCore.App.Internal package just for this purpose, but we're still fully planning on removing production of the Microsoft.NETCore.App RID-specific packages.

Reply:
Yeah, an empty package is fine (SDK does this)